src/ui/tests/orders/checkUI/orderModals.spec.ts

```diff
@@ -0,0 +1,214 @@
+      await expect(selectManagerModal.getModalTitle()).resolves.toBe(
+        'Assign Manager',
+      );
+      await expect(selectManagerModal.managerSearchInput).toBeVisible();
+
+      await selectManagerModal.fillManagerSearchInput(
+        MOCK_MANAGER_OLGA.firstName,
+      );
+
+      await expect(
+        selectManagerModal.getManagerListItem(
+          `${MOCK_MANAGER_OLGA.firstName} ${MOCK_MANAGER_OLGA.lastName}`,
+        ),
+      ).toBeVisible();
+
+      await selectManagerModal.clickManagerListItem(
+        `${MOCK_MANAGER_OLGA.firstName} ${MOCK_MANAGER_OLGA.lastName}`,
+      );
+
+      await selectManagerModal.clickSaveButton();
+
+      await expect(selectManagerModal.modalTitle).not.toBeVisible();
+
+      const updatedOrderStatusTitle =
+        await orderDetailsPage.topPanel.getOrderDetailsPanelTitle();
+      await expect(updatedOrderStatusTitle).toBe(`Order Details`);
+      const assignedManagerName =
+        await orderDetailsPage.topPanel.getAssignedManagerName();
+      await expect(assignedManagerName).toBe(
+        `${MOCK_MANAGER_OLGA.firstName} ${MOCK_MANAGER_OLGA.lastName}`,
+      );
+    },
+  );
+});
+test.describe('[UI] [Orders] [Modals] [Remove Manager Modal]', () => {
+  let targetOrderId: string;
+  test.beforeEach(
+    async ({
+      homeUIService,
+      ordersPage,
+      orderDetailsPage,
+      orderManagerAssignedStatus,
+    }) => {
+      const PRODUCTS_TO_CREATE_COUNT = 1;
+      const { id } = await orderManagerAssignedStatus(PRODUCTS_TO_CREATE_COUNT);
+      targetOrderId = id;
+
+      await homeUIService.openAsLoggedInUser();
+      await homeUIService.openModule('Orders');
+      await ordersPage.waitForOpened();
+
+      await ordersPage.clickDetailsButton(targetOrderId);
+      await orderDetailsPage.topPanel.waitForOpened();
+    },
+  );
+
+  test(
+    'Should open "Remove Manager" modal and display its basic UI',
+    { tag: [TAGS.UI] },
+    async ({ confirmationModal, orderDetailsPage }) => {
+      await orderDetailsPage.topPanel.clickRemoveAssignedManagerButton();
+
+      await expect(confirmationModal.modalTitle).toBeVisible();
+      const modalTitle = await confirmationModal.getModalTitle();
+      await expect(modalTitle).toBe('Unassign Manager');
+      const modalContent = await confirmationModal.getModalContent();
+      await expect(modalContent).toContain(
+        'Are you sure you want to unassign manager from order?',
+      );
+      await expect(confirmationModal.confirmButton).toBeVisible();
+
+      await confirmationModal.clickConfirmButton();
+
+      await confirmationModal.modalContainer.waitFor({ state: 'hidden' });
+      await expect(confirmationModal.modalTitle).not.toBeVisible();
+
+      const updatedOrderStatusTitle =
+        await orderDetailsPage.topPanel.getOrderDetailsPanelTitle();
+      await expect(updatedOrderStatusTitle).toBe(`Order Details`);
+      await expect(orderDetailsPage.topPanel.assignManagerButton).toBeVisible();
```

Comment:
Можно делать отступы для читабельности
```suggestion
      await expect(confirmationModal.modalTitle).toBeVisible();
      
      const modalTitle = await confirmationModal.getModalTitle();
      
      await expect(modalTitle).toBe('Unassign Manager');
      
      const modalContent = await confirmationModal.getModalContent();
      
      await expect(modalContent).toContain(
        'Are you sure you want to unassign manager from order?',
      );
      await expect(confirmationModal.confirmButton).toBeVisible();

      await confirmationModal.clickConfirmButton();
      await confirmationModal.modalContainer.waitFor({ state: 'hidden' });
      
      await expect(confirmationModal.modalTitle).not.toBeVisible();
      
      const updatedOrderStatusTitle =
        await orderDetailsPage.topPanel.getOrderDetailsPanelTitle();
        
      await expect(updatedOrderStatusTitle).toBe(`Order Details`);
      await expect(orderDetailsPage.topPanel.assignManagerButton).toBeVisible();
```